examples/sops-secrets/README.md

```diff
@@ -0,0 +1,12 @@
+## sops-secrets
+
+**Note:** this example uses CircleCI v1
```

Comment:
We should probably open another issue to update all the rest of these to v2

Reply:
Done!